tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrdersBuilder.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders
+{
+    internal sealed class OrdersBuilder

```

Comment:
This class is only building one Order object, not a list of orders. Found it a little confusing.

Reply:
Rename to "OrderBuilder"?

Reply:
👍